main/src/cgeo/geocaching/storage/DocumentContentAccessor.java

```diff
@@ -107,10 +107,10 @@ private Uri createInternal(@NonNull final Folder folder, @NonNull final String n
         final String docName = createUniqueFilename(name, queryDir(folderUri, new String[]{ DocumentsContract.Document.COLUMN_DISPLAY_NAME }, c -> c.getString(0)));
         try (ContextLogger cLog = new ContextLogger("DocumentFolderAccessor.create %s: %s", folder, name)) {
             //Do NOT pass a mimeType. It will then be selected based on the file suffix.
-            return DocumentsContract.createDocument(getContext().getContentResolver(), folderUri, null, docName);
+            //Use an empty string for mimeType; null triggers an exception in API21 and API30 (funnily enough: noz in API29)
```

Comment:
/noz/not/

Reply:
Damn! I will fix this with the next bug finding...

Reply:
done